src/app/components/client/ExposureCard.tsx

```diff
@@ -278,3 +361,13 @@ export const ExposureCard = (props: ExposureCardProps) => {
 
   return exposureCard;
 };
+
+function formatOtherBreachCategoriesLabel(sentence: string) {
+  const words = sentence.split("-");
+  const formattedWords = words.map((word) => {
+    const firstLetter = word.charAt(0).toUpperCase();
+    const restOfWord = word.slice(1);
+    return firstLetter + restOfWord;
+  });
+  return formattedWords.join(" ");
+}
```

Comment:
Not sure if this is any clearer/better, but seemed to give the same results based on my limited testing:
```js
function formatOtherBreachCategoriesLabel(sentence) {
  return sentence
    .replaceAll("-", " ")
    .replace(/^[a-z]/, (i) => i.toUpperCase());
}
```

Reply:
@codemist Still curious what the method does, and why we need it.
Is it mainly for converting `DataClasses` slugs like "delivery-instructions" and "email-addresses" into "Delivery instructions" and "Email classes"?

All those should already be localized in https://github.com/mozilla/blurts-server/blob/main/locales/en/data-classes.ftl files.